service/frontend/adminHandler.go

```diff
@@ -227,6 +227,68 @@ func (adh *AdminHandler) AddSearchAttributes(ctx context.Context, request *admin
 	}, nil
 }
 
+// RemoveSearchAttributes add search attribute to the cluster.
+func (adh *AdminHandler) RemoveSearchAttributes(ctx context.Context, request *adminservice.RemoveSearchAttributesRequest) (_ *adminservice.RemoveSearchAttributesResponse, retError error) {
+	defer log.CapturePanic(adh.GetLogger(), &retError)
+
+	scope, sw := adh.startRequestProfile(metrics.AdminRemoveSearchAttributesScope)
+	defer sw.Stop()
+
+	// validate request
+	if request == nil {
+		return nil, adh.error(errRequestNotSet, scope)
+	}
+
+	if len(request.GetSearchAttributes()) == 0 {
+		return nil, adh.error(errSearchAttributesNotSet, scope)
+	}
+
+	indexName := request.GetIndexName()
+	if indexName == "" {
+		indexName = adh.ESConfig.GetVisibilityIndex()
+	}
```

Comment:
index name should be required?

Reply:
Index name is hidden now. I added it for future needs.